distributed/shuffle/multi_file.py

```diff
@@ -0,0 +1,272 @@
+import asyncio
```

Comment:
Testing on 3.8 and ran into this :) 

```suggestion
from __future__ import annotations

import asyncio
```

Reply:
Thanks @shwina !